src/uk/gov/defra/ffc/Utils.groovy

```diff
@@ -120,7 +120,7 @@ class Utils implements Serializable {
 
   static def getProvisionedQueueConfigValues(ctx, repoName, pr) {
     def configValues = [:]
-    def filePath = './provision.azure.yaml'
+    def filePath = Provision.azureProvisionConfigFile
```

Comment:
it makes me nervous having Provision use Utils methods, and Utils use Provision methods as it introduces a cyclic dependency.

As these methods have Provision in the name, would they be better off in the Provision class?

Reply:
Yup, good call.